### DISCUSSION TITLE
Invoice of GitHub

### DISCUSSION THREAD

**Discussion by swierczd828** [community/community#54964] [Other Feature Feedback, Questions, & Ideas] [answered]
Dear all
My organization paid for GitHub  Team version.
We didn't recive any invoice.
Do you know how solve this issue?
Do you know the e-mail or phone to financial department?

**Comment by ryanlua** [comment, marked as answer]
Hi @swierczd828,

You can contact GitHub support by filling out a [support request](https://support.github.com/request) and selecting <kbd>Account or billing issues</kbd>.

For you invoice, it will be sent to the verified primary email address of the GitHub account that purchased it or the email address you entered when you purchased it. If it is not there than you can contact GitHub support from the above link for help with your issue.

**Reply by 09911105935** [reply]
دریافت مالی را تقاضا دارم لطفا حمایت کنید 
Mohamabjvadrezvani@gmail.com 
Rezvania138@gmail.com 
09138563430

**Reply by 09911105935** [reply]
▶️800000000#$$# HEXN’s 🪙 100,000%78900000@ USDT + *️⃣/÷789032000/$%$# 1,000,000 HP Referral Contest Is Still LIVE09138563430 Contest Details: •Gorg1365gorg....mkd8000000000.6037998105062144@#Starting Prize Pool:؟؟ 🪙 20,000 USDT *️⃣ 200,000 HP • Pool2؟ Growth: 🪙 +2,000 USDT *️⃣ +20,000 HP per 1MLN new users • Max Priz

**Reply by 09911105935** [reply]
مبلغی حدود 8000000000میلیون نیاز دارم ممنون میشم 
کمکم کنید ....6037998105062144@$$%/$$

**Reply by RioAlfarizi321** [reply]
/

**Reply by Issalk** [reply]
issaldonn@gmail.comأنا

**Reply by Soegyimc** [reply]
Hi pls help me

**Comment by legoutech** [comment]
Can you provide me with an invoice with payment information?
Our information includes:
Company Name: Infinite Interactive Limited
Address: The Grand Pavilion Commercial centre, Oleander Way, 802 West Bay Road, P.O. Box 32052, Grand Cayman KY1-1208, Cayman Islands
Receiving email: admin@legoutech.com

**Reply by ryanlua** [reply]
Hi @legoutech,

I don't work for GitHub and this is a community run so we cannot provide you with an invoice. You should contact GitHub as mentioned in the answer.

Additionally you shouldn't share any sensitive or private information as anyone can view it here.

**Comment by alpenfacilities** [comment]
Good Morning,
I receive the receipt of payment, but for the accounting department prefer to have the invoice. Please send me the invoice to: facilities@arsenalia.group. 
Thanks in advance.

Massimo Salviato
+39 3405928764

**Comment by MarinaRosso** [comment]
Good morning,

Can you send me the invoice of september 2024?

**Reply by djmartberry66** [reply]
Nice

**Comment by it-golden2024** [comment]
Hi Github,
I have already registered the enterprise github plan.
Why in my github is still on free github plan?
Please help check it!
Your order for GOLDEN LOTUS VIETNAM COMPANY LIMITED 10 Seat 2025_Q3 Renewal has been processed
(PO#: PO2503000791).
The license start date is 3/25/2025 and the end date is 3/24/2026

**Reply by MidoElshraihy** [reply]
https://WWW.Example/proxy.com.Pac

**Reply by it-golden2024** [reply]
Hi MidoEIshraihy,
I can't access that link.
I don't understand what you mean? please explain how to resolve my issue?

**Comment by harbans2587** [comment]
Hi

**Comment by duynalit** [comment]
![image](https://github.com/user-attachments/assets/1342ee74-7731-43e6-9002-b2ad8f4f6a01)
![Capture](https://github.com/user-attachments/assets/1135ee5b-b303-4c14-9a8d-ebbda6ea36fe)
I have updated the upgrade for $4 more but during the process of using requests from visual studio code at Cloud Sonnet 4, I am still charged for each request. Please help me update my package.

**Comment by Raphasilva2025** [comment]
Entao eu adquiri o github pro pelo app dele no celular mas eu continuo no plano gratuito o que eu faço 
![WhatsApp Image 2025-08-18 at 15 42 08](https://github.com/user-attachments/assets/397d1a73-b50d-4682-b6c5-324a0b911793)
nao estou conseguido usar ele no visual code.

**Comment by 1ca32b** [comment]
What is your contact email? Can we push it through so, to understand more communication with?

**Comment by hmadhkhwjly3-png** [comment]
> Dear all My organization paid for GitHub Team version. We didn't recive any invoice. Do you know how solve this issue? Do you know the e-mail or phone to financial department?

**Comment by tojin12341za** [comment]
email :tojin12341za@gmail.com

**Comment by mochrira** [comment]
Can i enable github copilot pro again, after automatically switched back to free because of failed payment?

**Comment by swierczd828** [comment]
We will pay for the subscription.
Our process is going soon.

[Logo Ł-IEL]<https://iel.lukasiewicz.gov.pl/>

dr inż. Dariusz Świerczyński
Lider obszaru
Grupa Badawcza Napędów Elektrycznych i Sterowania
Centrum Badawcze Elektromobilności i Inteligentnego Transportu

[cid:3e0814f8-3532-4bb8-9751-5ae8a43f7103]  ***@***.******@***.***>
[cid:1ed25a3e-c77f-4518-a38a-a969a9e88aee]  +48 601 960 244
[cid:c6e54e07-fefd-44c4-a2e8-230599e12fd2]  +48 22 11 25 273
[cid:eff856f8-91c1-49f1-aa07-0ff8691de6a1]  Teams ***@***.***&type=chat>
Sieć Badawcza Łukasiewicz - Instytut Elektrotechniki
ul. Mieczysława Pożaryskiego 28, 04-703 Warszawa, NIP: 5250007684
[cid:38e2faea-8ab9-44a0-8d42-a28ab45fbefc]<https://pl.linkedin.com/company/instytut-elektrotechniki>  [cid:9dfe63b6-9522-4327-947b-57dfa3fba49a] <https://www.facebook.com/Sie%C4%87-Badawcza-%C5%81ukasiewicz-Instytut-Elektrotechniki-2043146522467933/>
Sąd Rejonowy dla m.st. Warszawy w Warszawie, XIV Wydz. Gospodarczy KRS nr 0000857831

Administratorem danych osobowych jest Dyrektor Sieć Badawcza Łukasiewicz - Instytutu Elektrotechniki, z siedzibą przy ul. Mieczysława Pożaryskiego 28, 04-703 Warszawa. Kontakt z inspektorem ochrony danych osobowych dostępny jest poprzez adres e-mail: ***@***.******@***.***>. Szczegółowe klauzule informacyjne zamieszczone zostały pod adresem:
https://iel.lukasiewicz.gov.pl/dane-osobowe/

________________________________
Od: Moch. Rizal Rachmadani ***@***.***>
Wysłane: czwartek, 14 maja 2026 06:08
Do: community/community ***@***.***>
DW: Dariusz Świerczyński | Łukasiewicz – IEL ***@***.***>; Mention ***@***.***>
Temat: Re: [community/community] Invoice of GitHub (Discussion #54964)


** OSTRZEŻENIE systemu pocztowego Łukasiewicza **
E-Mail pochodzi spoza organizacji. Nie klikaj łączy ani nie otwieraj załączników, chyba że rozpoznajesz nadawcę i wiesz, że zawartość jest bezpieczna.
.

Can i enable github copilot pro again, after automatically switched back to free because of failed payment?

—
Reply to this email directly, view it on GitHub<https://github.com/orgs/community/discussions/54964#discussioncomment-16912734>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AWGNSNOLGOVEPHOGTNFGVPD42VBFRAVCNFSM6AAAAABNJXUAYWVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTMOJRGI3TGNA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by gold09309-max** [comment]
> Dear all My organization paid for GitHub Team version. We didn't recive any invoice. Do you know how solve this issue? Do you know the e-mail or phone to financial department?

**Comment by gold09309-max** [comment]
https://github.com/orgs/community/discussions/54964#discussion-5178230

**Comment by gold09309-max** [comment]
Millions of naira
